setup.py

```diff
@@ -31,7 +31,6 @@
         "googleapis-common-protos>=1.57",
         "flyteidl>=1.10.0",
         "wheel>=0.30.0,<1.0.0",
-        "pandas>=1.0.0,<2.0.0",
         "pyarrow>=4.0.0,<11.0.0",
```

Comment:
Just as a reminder. This can also be deleted.

Reply:
done

Reply:
It looks like it was added back to the dependencies in pyproject.toml? Is that intended? Sorry I'm not following up the PR closely.